3.10/graphs-pregel.md

```diff
@@ -103,14 +98,13 @@ The `state` field has one of the following values:
 | State          | Description    |
 |:---------------|:---------------|
 | `"running"`    | Algorithm is executing normally.
-| `"in error"`   | The execution is in an error state. This can be caused by primary DB-Servers being not reachable or being non responsive. The execution might recover later, or switch to "canceled" if it was not able to recover successfully
```

Comment:
@Simran-B Julia has added three more states (loading, fatal_error, none). Looks like they were removed during your review. Please add them back.

Reply:
Sorry @jvolmer, the visual 3-way merge in VS Code confused me big time. Restored.